addons/website_sale/views/templates.xml

```diff
@@ -242,7 +242,7 @@
         <div t-attf-class="o_pricelist_dropdown dropdown#{'' if website_sale_pricelists and len(website_sale_pricelists)&gt;1 else ' d-none'} #{_classes}">
             <t t-set="curr_pl" t-value="website.get_current_pricelist()" />
             <span class="font-weight-bold text-muted">Pricelist:</span>
-            <a role="button" href="#" class="dropdown-toggle btn btn-light border-0 px-0 text-muted align-baseline" data-toggle="dropdown">
+            <a role="button" href="#" class="dropdown-toggle btn btn-light border-0 px-0 text-muted align-baseline" data-toggle="dropdown" data-display="static">
```

Comment:
By doing this, you create other bugs, like this one (especially true in mobile):
![Screenshot from 2021-04-28 10-33-30](https://user-images.githubusercontent.com/10338094/116373035-46c09700-a80d-11eb-966f-ed600ec0fa4e.png)

I guess this could be fixed by aligning the dropdown-menu on the right with the `dropdown-menu-right` class but I wonder if the current "bug" is a real one worth fixing at all? And if needed, is not it to be fixed in stable versions as well?

Reply:
Hi, @qsm-odoo 
Thanks for inputs. :)
indeed this fixed issue (https://drive.google.com/file/d/1rCpQAF4ZsLsMDKC_TIW5swHBK-tCw-JF/view?usp=sharing)

please check the below screenshot for current bug,
https://drive.google.com/file/d/1z5v0kFzzSoBQpuQWU5f2lagx_5x0SQY_/view?usp=sharing
when list is too long due to dynamic postioining of dropdown. it attaches to top of page (above pricelist button).

Yes, this is problem from the version 12. so, to be fixed in stable versions.
But as the change is made in python template I have targeted master.